<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QWidget" name="Layout">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>Best Chromosome:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>CrossOver Rate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>Population:</string>
      </property>
      <property name="buddy">
       <cstring>text_pop</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>Genome:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="text_mutation">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(35, 16, 223);</string>
      </property>
      <property name="text">
       <string>0.2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="text_pop">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(35, 16, 223);</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="text_elitism">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(35, 16, 223);</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>Mutation Rate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
      </property>
      <property name="text">
       <string>Generations:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="text_generation">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(35, 16, 223);</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLineEdit" name="text_crossRate">
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="10">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>433</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="10">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;</string>
        </property>
        <property name="text">
         <string>Algorithm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="box_algorithm">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(118, 92, 92);</string>
        </property>
        <item>
         <property name="text">
          <string>Fire Station</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NQueens</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>K-Means</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="10">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;
border-color: rgb(0, 0, 0);
background-color: rgb(137, 118, 118);</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;
background-color: rgb(137, 118, 118);</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_reset">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Century Schoolbook L&quot;;
background-color: rgb(137, 118, 118);</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="10">
     <widget class="QCustomPlot" name="graphInterface" native="true"/>
    </item>
    <item row="5" column="1" colspan="9">
     <widget class="QLineEdit" name="text_answer"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>text_pop</tabstop>
  <tabstop>text_generation</tabstop>
  <tabstop>text_elitism</tabstop>
  <tabstop>text_mutation</tabstop>
  <tabstop>text_crossRate</tabstop>
  <tabstop>button_start</tabstop>
  <tabstop>button_reset</tabstop>
  <tabstop>button_stop</tabstop>
  <tabstop>box_algorithm</tabstop>
  <tabstop>text_answer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
